code/datums/components/storage/concrete/pockets.dm

```diff
@@ -50,7 +50,7 @@
 	if(istype(I,/obj/item/clothing/head/mob_holder))
 		var/obj/item/clothing/head/mob_holder/mausholder = I
 		if(locate(/mob/living/simple_animal/mouse) in mausholder.contents)
-			return TRUE
+			return .
 		return FALSE
```

Comment:
and this should just be a return?

Reply:
Does "return" with no "." after it return the value of ".", or just FALSE?

Reply:
if this returns just ., then mouse-less mob_holders (like, uh, drones and stuff, I guess) will be able to fit inside of the hat